--- trunk/sweetyRP/phone_system.lua
+++ trunk/sweetyRP/phone_system.lua
@@ -1,21 +1,21 @@
-﻿calltimer={}
+ calltimer={}
 
 function StartCall( source, cmd, nomer )
 	if tonumber(getElementData ( source, "phone" )) > 0 then
 		for id, player in ipairs ( getElementsByType ( "player" ) ) do
 			if getElementData ( source, "phone" ) == nomer then
 			--	if getElementData ( source, "phoneturn" ) == 1 then
-					outputChatBox (" (Телефон) Звонок прошел, ждите", source, 204,204,255 )
-					outputChatBox (" Вам звонит абонент "..tostring(getElementData ( source, "phone" ))..". Что бы взять трубку - введите /acall ", player, 204,204,255 )
-					if ( playerToPoint ( player, 5, x, y, z ) ) then outputChatBox ( " У кого то звонит телефон ", player, 51,204,255 ) end
+        outputChatBox (" (Phone) You are connected, wait", source, 204,204,255 )
+        outputChatBox (" Incoming call from "..tostring(getElementData ( source, "phone" ))..". Type /acall to answer ", player, 204,204,255 )
+        if ( playerToPoint ( player, 5, x, y, z ) ) then outputChatBox ( " Somebody's phone is ringing ", player, 51,204,255 ) end
 					calltimer[tonumber(getElementData ( source, "phone" ))] = setTimer ( breakCall, 10000, 1, source, player )
 					setElementData ( source, "call", getElementData ( player, "playerid" ) )
 					setElementData ( player, "call", getElementData ( source, "playerid" ) )
 			--	else
-					outputChatBox (" (Телефон) Аппарат абонента выключен", source, 204,204,255 )
+        outputChatBox (" (Phone) The phone is turned off", source, 204,204,255 )
 			--	end
 			else
-				outputChatBox (" (Телефон) Неправильно набран номер", source, 204,204,255 )
+        outputChatBox (" (Phone) Wrong telephone number", source, 204,204,255 )
 			end
 		end
 	end
@@ -24,13 +24,13 @@
 addCommandHandler ("call", StartCall)
 
 function AcceptCall ( source )
-	outputChatBox ("Информация: Вы подняли трубку",source, 0,153,51 )
+  outputChatBox ("Info: You answered the call",source, 0,153,51 )
 	for id, player in ipairs ( getElementsByType ( "player" ) ) do
 		if getElementData ( source, "call" ) == getElementData ( player, "playerid" ) then
 			if getElementData ( player, "call" ) == getElementData ( source, "playerid" ) then
 				setElementData ( source, "ocall", getElementData ( source, "playerid" ) )
 				setElementData ( player, "ocall", getElementData ( player, "playerid" ) )
-				outputChatBox (" (Телефон) Вам ответили, говорите", player, 204,204,255 )
+        outputChatBox (" (Phone) Your call is answered, speak", player, 204,204,255 )
 				killTimer ( calltimer[tonumber(getElementData ( source, "phone" ))] )
 			end
 		end
@@ -39,8 +39,8 @@
 addCommandHandler ("acall", AcceptCall)
 
 function breakCall ( source, player )
-	outputChatBox (" (Телефон) Абонент не отвечает", source, 204,204,255 )
-	outputChatBox (" (Телефон) Звонок сбошен", player, 204,204,255 )
+  outputChatBox (" (Phone) No answer", source, 204,204,255 )
+  outputChatBox (" (Phone) The call was rejected", player, 204,204,255 )
 	setElementData ( source, "call", "0" )
 	setElementData ( player, "call", "0" )
 	giveBizMoney( 12, callprice, 1 )
@@ -51,13 +51,13 @@
 function HangUpCall( source, cmd )
 	setElementData ( source, "call", "0" )
 	setElementData ( source, "ocall", "0" )
-	outputChatBox (" (Телефон) Вы положили трубку", source, 204,204,255 )
+  outputChatBox (" (Phone) You rejected the call", source, 204,204,255 )
 	for id, player in ipairs ( getElementsByType ( "player" ) ) do
 		if getElementData ( source, "ocall" ) == getElementData ( player, "playerid" ) then
 			if getElementData ( player, "ocall" ) == getElementData ( source, "playerid" ) then
 			setElementData ( player, "call", "0" )
 			setElementData ( player, "ocall", "0" )
-			outputChatBox (" (Телефон) Ваш собеседник положил трубку", player, 204,204,255 )
+        outputChatBox (" (Phone) Your interlocutor rejected the call", player, 204,204,255 )
 			end
 		end
 	end
@@ -77,27 +77,27 @@
 				for id, player in ipairs ( getElementsByType ( "player" ) ) do
 					if ( tonumber(getElementData ( player, "phone" )) ==  tonumber(ident) ) then
 						if(getElementData ( source, "tied" ) ~= 1) then
-					    	outputChatBox ("СМС от " .. getPlayerName(player) .. ": "..text.." ",player, 255,255,127 )
+              outputChatBox ("SMS from " .. getPlayerName(player) .. ": "..text.." ",player, 255,255,127 )
 					    	local x,y,z = getElementPosition( source )
-					    	sendActionMessage( source, x, y, z, 6, "пишет смс" )
+              sendActionMessage( source, x, y, z, 6, "is writing an SMS" )
 					    	setElementData ( source, "pmoney", playerpmoney - smsprice )
 					    	playeronline = 1
 		                else
-		                    outputChatBox ("Вас связали!",source, 255,255,127 )
+              outputChatBox ("You are connected!",source, 255,255,127 )
 						end
 					end
 				end
 				if (playeronline == 0) then
-					outputChatBox (" Этот игрок не в сети ",source, 255,255,127 )
+          outputChatBox (" This player is offline ",source, 255,255,127 )
 				end
 			else
-				outputChatBox ("Невозможно отправить сообщение: не хватает денег на счету",source, 255,255,127 )
+        outputChatBox ("Impossible to send the message: haven't enough money",source, 255,255,127 )
 			end
 		else
-			outputChatBox ("Сначала купите телефон в 24/7",source, 255,255,127 )
+      outputChatBox ("Buy yourself a phone first (at one of the 24/7 stores)",source, 255,255,127 )
 		end
 	else
-		outputChatBox ("Правильно: /sms [номер телефона] текст ", 255,255,127 )
+    outputChatBox ("Correct: /sms [phone number] text ", 255,255,127 )
 	end
 end
 
