--- trunk/sweetyRP/client/pdd.lua
+++ trunk/sweetyRP/client/pdd.lua
@@ -1,4 +1,4 @@
-﻿localPlayer = getLocalPlayer ()
+ localPlayer = getLocalPlayer ()
 
 GUIEditor_Memo = {}
 GUIEditor_Window = {}
@@ -12,9 +12,9 @@
 
 function instructionPDDMemo ()
 	GUIEditor_Window[18] = guiCreateWindow(495,303,508,331,"Rules",false)
-	GUIEditor_Memo[2] = guiCreateMemo(17,23,482,228,"Max speed in city - 70 km/h. Max Speed outside - 100 km/h. Left. See the full rules at our site - jesw.co.cc",false,GUIEditor_Window[18])
+  GUIEditor_Memo[2] = guiCreateMemo(17,23,482,228,"Max speed in the city - 70 km/h. Max Speed outside - 100 km/h. Left. See the full rules on our website - jesw.co.cc",false,GUIEditor_Window[18])
 	guiMemoSetReadOnly(GUIEditor_Memo[1],true)
-	GUIEditor_Checkbox[1] = guiCreateCheckBox(159,259,175,28,"I read this rules",false,false,GUIEditor_Window[18])
+  GUIEditor_Checkbox[1] = guiCreateCheckBox(159,259,175,28,"I read these rules",false,false,GUIEditor_Window[18])
 	GUIEditor_Button[90] = guiCreateButton(161,290,141,24,"Next",false,GUIEditor_Window[18])
 	guiSetVisible ( GUIEditor_Window[18], true )
 	showCursor ( true )
@@ -32,9 +32,9 @@
 addEventHandler( "instructionPDDMemoShow", getRootElement(), instructionPDDMemo )
 
 function instructionPDDTest()
-	outputChatBox ("Подсказка: Выберите три павильных ответа", 102,204,153 )
+  outputChatBox ("Hint: Choose three correct answers", 102,204,153 )
 	GUIEditor_Window[19] = guiCreateWindow(528,265,380,389,"Test",false)
-	GUIEditor_Label[36] = guiCreateLabel(78,94,266,20,"Max speed in city?",false,GUIEditor_Window[19])
+  GUIEditor_Label[36] = guiCreateLabel(78,94,266,20,"Max speed in the city?",false,GUIEditor_Window[19])
 	guiLabelSetColor(GUIEditor_Label[36],255,255,255)
 	guiLabelSetVerticalAlign(GUIEditor_Label[36],"top")
 	guiLabelSetHorizontalAlign(GUIEditor_Label[36],"left",false)
@@ -47,7 +47,7 @@
 	GUIEditor_Checkbox[2] = guiCreateCheckBox(30,123,75,21,"70 kmph",false,false,GUIEditor_Window[19])
 	GUIEditor_Checkbox[3] = guiCreateCheckBox(149,123,75,21,"150 kmph",false,false,GUIEditor_Window[19])
 	GUIEditor_Checkbox[4] = guiCreateCheckBox(265,123,75,21,"100 kmph",false,false,GUIEditor_Window[19])
-	GUIEditor_Label[38] = guiCreateLabel(78,171,266,15,"Max speed in outside?",false,GUIEditor_Window[19])
+  GUIEditor_Label[38] = guiCreateLabel(78,171,266,15,"Max speed in the outside?",false,GUIEditor_Window[19])
 	guiLabelSetColor(GUIEditor_Label[38],255,255,255)
 	guiLabelSetVerticalAlign(GUIEditor_Label[3],"top")
 	guiLabelSetHorizontalAlign(GUIEditor_Label[38],"left",false)
@@ -89,7 +89,7 @@
 		    if guiCheckBoxGetSelected ( GUIEditor_Checkbox[2] ) and guiCheckBoxGetSelected ( GUIEditor_Checkbox[7] ) and guiCheckBoxGetSelected ( GUIEditor_Checkbox[8] ) then
 				instructionPDDFin()
       		else
-      			outputChatBox("Информация: Тест провален!", 0,153,51)
+        outputChatBox("Info: You failed the test!", 0,153,51)
 				showCursor ( false )
 			end
 			guiSetVisible ( GUIEditor_Window[19], false )
@@ -99,7 +99,7 @@
 
 function instructionPDDFin ()
 	GUIEditor_Window[20] = guiCreateWindow(566,433,378,128,"Congratulations!",false)
-	GUIEditor_Label[43] = guiCreateLabel(31,31,335,20,"Your test is succes! Now you must to drive a real car",false,GUIEditor_Window[20])
+  GUIEditor_Label[43] = guiCreateLabel(31,31,335,20,"Your succeeded! Now you can drive a real car",false,GUIEditor_Window[20])
 	guiLabelSetColor(GUIEditor_Label[43],255,255,255)
 	guiLabelSetVerticalAlign(GUIEditor_Label[43],"top")
 	guiLabelSetHorizontalAlign(GUIEditor_Label[43],"left",false)
@@ -108,7 +108,7 @@
 	guiSetVisible ( GUIEditor_Window[20], true )
 	addEventHandler("onClientGUIClick", GUIEditor_Button[77],
 		function ()
-		   	outputChatBox ("Подсказка: Едьте по чекпоинтам, не привышайте 70 км/ч и не испорьте машину!", 102,204,153 )
+      outputChatBox ("Hint: Drive through the checkpoints, don't drive more than 70 km/h and don't damage the car!", 102,204,153 )
 		   	--carfortest = createVehicle ( 566, 1148.64453125,1375.7490234375,10.419050216675 )
 			guiSetVisible ( GUIEditor_Window[20], false )
 			test = createMarker ( 1058.8583984375,1375.248046875,10.446697235107, "checkpoint" )
@@ -183,7 +183,7 @@
   		toggleControl ( "enter_exit", true )
 		setElementData ( localPlayer, "cartest", 0 )
 		setElementPosition ( localPlayer, 1168.443359375,1364.6494140625,10.812507629395 )
-		outputChatBox("Информация: Поздравляю! Вы успешно выполнили тест на права, и теперь можете водить машину", 0,153,51)
+    outputChatBox("Info: Congratulations! You passed the test successfully and now car drive a car", 0,153,51)
 		removeEventHandler( "onClientMarkerHit", test, MarkerHit )
 		triggerServerEvent ( "onGreetingFinish", getLocalPlayer () )
   		setElementData ( localPlayer, "drive", 1)
